sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/ResourceManager.java

```diff
@@ -164,7 +164,8 @@ public static Configurable configure() {
             this.sdkContext = new SdkContext();
             this.subscriptionClient = new SubscriptionClientBuilder()
                     .pipeline(httpPipeline)
-                    .endpoint(profile.environment().getResourceManagerEndpoint())
+                    .environment(profile.getEnvironment())
```

Comment:
AzureEnvionment is missed for all service client builder. Just add for each one.

Reply:
No harm but actually it is not used inside client.

It was there at day-1, and I kind of "maybe one day client needs it".